.github/workflows/check-docs-links.yaml

```diff
@@ -0,0 +1,19 @@
+on: [push, pull_request]
+name: Check docs links
+jobs:
+  check-docs-links:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Link Checker
+        id: lc
+        uses: peter-evans/link-checker@v1
+        with:
+          args: >
+            --concurrency 3
+            --document-root "https://loopback.io/"
```

Comment:
do we need `--document-root` if we've specified `docs/**/*.md` below?

Reply:
`--document-root` is used for absolute paths. In previous commits, there were errors with some links that follow this style: `/docs/en/...` due to the document root not being set.